cluster/gce/upgrade-aliases.sh

```diff
@@ -35,9 +35,10 @@ source "${KUBE_ROOT}/cluster/kube-util.sh"
 # Assumed vars:
 #   PROJECT
 function get-k8s-node-routes-count() {
-  local k8s_node_routes_count=$(gcloud compute routes list \
-    --project=${PROJECT} --filter='description=k8s-node-route' \
+  k8s_node_routes_count=$(gcloud compute routes list \
+    --project="${PROJECT}" --filter='description=k8s-node-route' \
     --format='value(name)' | wc -l)
+  local k8s_node_routes_count
```

Comment:
why change: 
```
In ./cluster/gce/upgrade-aliases.sh line 38:
  local k8s_node_routes_count=$(gcloud compute routes list \
        ^-------------------^ SC2155: Declare and assign separately to avoid masking return values.
```
Ref: https://github.com/koalaman/shellcheck/wiki/SC2155